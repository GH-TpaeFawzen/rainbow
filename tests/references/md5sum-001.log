0bc2ee1037199736ad2b8b144f7834f1  setup.py
