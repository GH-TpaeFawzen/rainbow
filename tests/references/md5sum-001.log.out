[FAINT]0bc2ee1037199736ad2b8b144f7834f1[RESET_INTENSITY]  setup.py
